addons/mod_tool/dock.gd

```diff
@@ -2,8 +2,6 @@ tool
 extends Control
 
 
-const ERROR_COLOR = "#ff9090"
```

Comment:
We can also get that color from the theme in `set_editor_interface `. More consistent if the user has a different editor theme
```gdscript
base_theme = editor_interface.get_base_control().theme
store.error_color = base_theme.get_color("error_color", "Editor")
```

Reply:
✅ [ac28aa3](https://github.com/GodotModding/godot-mod-tool/pull/20/commits/ac28aa3143e38d1fa7879df5e7d43aea785a3560)